cli/cmd/compliance_gcp.go

```diff
@@ -48,14 +48,34 @@ var (
 				return errors.Wrap(err, "unable to list gcp projects/organizations")
 			}
 
+			if len(response.Data) == 0 {
+				msg := `There are no GCP integrations configured in your account.
+
+Get started by integrating your GCP to analyze configuration compliance using the command:
+
+    $ lacework integration create
+
+If you prefer to configure the integration via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Settings > Integrations > Cloud Accounts.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+
 			gcpAccounts := extractGcpAccounts(response)
 
 			for _, gcp := range gcpAccounts {
 				rows = append(rows, []string{gcp.OrganizationID, gcp.ProjectID})
 			}
 
 			if cli.JSONOutput() {
-				return cli.OutputJSON(gcpAccounts)
+				jsonOut := struct {
+					Accounts []gcpAccount `json:"gcp_accounts"`
```

Comment:
I don't think Google has the concept of "accounts". They have either Organizations, projects, or folders.